src/controllers/employer.ts

```diff
@@ -65,6 +65,44 @@ export const registerEmployer = async (
   }
 }
 
+//**Update employer*/
+export const updateEmployer = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  try {
+    const employerId = req.params.id
+    const { update } = req.body
+    const employer = await Employer.findOne({ where: { id: employerId } })
+
+    if (!employer) {
+      throw new Error(`Employer ${employerId} not found`)
+    }
+    if (update.companyName) {
+      employer.companyName = update.companyName
+    }
+    if (update.companyInfo) {
+      employer.companyInfo = update.companyInfo
+    }
+    if (update.address) {
+      employer.address = update.address
+    }
+    if (update.email) {
+      employer.credentials.email = update.email
+    }
+    if (update.password) {
+      employer.credentials.password = update.password
+    }
+    const updatedEmployer = await Employer.update(employerId, update)
+    res.json(updatedEmployer)
+    res.status(200).json({ message: 'Updated successfully' })
```

Comment:
Why 2 `res.json()`here. you can combine them:
`res.json({message: 'success', data: updatedEmployer})`